yarn.lock

```diff
@@ -11083,12 +11088,13 @@ kolibri-components@0.13.0-dev.9:
     popper.js "^1.14.6"
     purecss "^0.6.2"
 
-"kolibri-design-system@https://github.com/learningequality/kolibri-design-system#develop":
-  version "0.2.0"
-  resolved "https://github.com/learningequality/kolibri-design-system#7d9829c7456819e6aa1e31f9ba939d1a5634d3d9"
+"kolibri-design-system@https://github.com/learningequality/kolibri-design-system#v0.2.0":
+  version "0.2.0-beta6"
```

Comment:
whoops, just noticed the version is still set to a beta here: 

https://github.com/learningequality/kolibri-design-system/blob/v0.2.x/package.json#L2

oh well, no big deal I don't think

Reply:
fixed this